packages/now-cli/src/types.ts

```diff
@@ -119,7 +110,6 @@ export type NpmDeployment = {
   created: number;
   creator: { uid: string };
   sessionAffinity: string;
-  scale: DeploymentScale;
 };
 
 export type StaticDeployment = {
```

Comment:
@TooTallNate Is `StaticDeployment` still used?

Reply:
I think I can merge `NpmDeployment` and `StaticDeployment` into just `Deployment` now?

Reply:
Saw your above comment